components/Events/index.js

```diff
@@ -79,4 +80,6 @@ const StyledDetails = styled.div`
   float: left;
```

Comment:
Is there any particular reason why you're using floats? I'd very much rather you didn't (ever), if it can be avoided. 😂

Reply:
Float is gone! :)